requirements.txt

```diff
@@ -1,4 +1,5 @@
 alabaster==0.7.10
+appnope==0.1.0
```

Comment:
what's this one?

Reply:
When I installed those four libs, appnote was automately added.